repos/spack_repo/builtin/packages/rocm_device_libs/package.py

```diff
@@ -25,6 +25,7 @@ def url_for_version(self, version):
 
     maintainers("srekolam", "renjithravindrankannath", "haampie", "afzpatel")
 
+    version("7.0.0", sha256="3d479a2aa615b6bb35cd3521122fbff34188dc0cc52d8b0acda59f9f55198211")
```

Comment:
update the homepage and git to https://github.com/ROCm/llvm-project and https://github.com/ROCm/llvm-project.git